src/ngraph/runtime/cpu/builder/one_hot.cpp

```diff
@@ -47,7 +47,8 @@ namespace ngraph
                     std::function<decltype(runtime::cpu::kernel::one_hot_rank_0<float>)> kernel;
                     SELECT_KERNEL(
                         kernel, out[0].get_element_type(), runtime::cpu::kernel::one_hot_rank_0);
-                    auto functor = [&, kernel, out_shape, one_hot_axis](CPURuntimeContext* ctx) {
+                    auto functor = [&, kernel, out_shape, one_hot_axis](CPURuntimeContext* ctx,
+                                                                        CPUExecutionContext* ectx) {
                         kernel(arg_tensor, out_tensor, out_shape, one_hot_axis);
```

Comment:
@jbobba : I am trying to understand: when do you pass `ectx->arena` to a kernel and when not? Eg: the kernel calls in functor of dot.cpp have a new argument `ectx->arena` .